.github/workflows/trigger-builds.yaml

```diff
@@ -0,0 +1,47 @@
+name: Trigger workflow for release
+
+on:
+  workflow_call:
+    inputs:
+      cache-manager-ref:
+        type: string
+        default: main
+      cache-manager-repository:
+        type: string
+        default: gingersnap-project/cache-manager
+      db-syncer-ref:
+        type: string
+        default: main
+      db-syncer-repository:
+        type: string
+        default: pminz/db-syncer
+      operator-ref:
+        type: string
+        default: main
+      operator-repository:
+        type: string
+        default: gingersnap-project/operator
+
+jobs:
+
+  cache-manager-images:
+    uses: gingersnap-project/cache-manager/.github/workflows/create-tag.yml@main
+    with:
+      ref: ${{ inputs.cache-manager-ref }}
+      repo: ${{ inputs.cache-manager-repository }}
+
+  db-syncer-image:
+    uses: gingersnap-project/db-syncer/.github/workflows/create-tag.yml@main
+    with:
+      ref: ${{ inputs.db-syncer-ref }}
+      repo: ${{ inputs.db-syncer-repository }}
+  
+  operator: 
+    if: success()
```

Comment:
Do we need this, I think `needs` is sufficient as this ensures the "operator" job won't be executed if either of those jobs fails.